shotover-proxy/examples/cassandra-passthrough/docker-entrypoint.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
this file looks like we've taken it from the docker image we pull in. Do you want to check whether we need to add anything to our NOTICES file from a license perspective?

Reply:
@justinmclean

Reply:
Added the changes to LICENSE